refarch-integrations/refarch-email-integration/refarch-email-integration-core/src/test/java/de/muenchen/refarch/email/integration/adapter/out/s3/S3AdapterTest.java

```diff
@@ -0,0 +1,76 @@
+package de.muenchen.refarch.email.integration.adapter.out.s3;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
Dont use Wildcard imports. Also remove them on other places. Try to also set it inside your IDE, so you can fix it faster and it doesnt happen again.